### PR TITLE
Created Native Installer script

### PR DESCRIPTION
**Pull request by joshpetit**
I followed the instructions found in #1 and made a script to automate the process. I used the 7z tool too extra the .exe (I'm not sure there's an alternative preinstalled on most systems) so it requires that to be installed, but there is a check at the start to ensure it's on the system.

I also updated the README to indicate these changes.

### CODE REVIEW

**Comment by Vedaant-Rajoo** [comment]
@puneetsl Kindly merge this request ASAP.

**Comment by AkechiShiro** [comment]
Thanks I tested it the script worked for me I'm on PopOS 20.04. This native installing script is only to extract the .exe (and create a shortcut for the Notion native app) so I take it it's not really made for Linux use ?? Or am I wrong ?

**Comment by joshpetit** [comment]
> so I take it it's not really made for Linux use ?? Or am I wrong ?

The script extracts the electron ASAR for windows. It will be just like running the windows notion app but on Linux. There shouldn't be any complications, but you're right, it wasn't made with Linux use in mind.

**Comment by AkechiShiro** [comment]
Okay thanks, so in that case to run the native app the binary to be used is the one contained in the `/Lotion` of the script I suppose.

**Comment by puneetsl** [comment]
Hey, thanks for contributing! 
And apologies, I was out for the weekend and couldn't take a look at it earlier. 
Let me do a quick code review.

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (puneetsl) [inline .idea/.gitignore:1]: Can we add `.idea` folder to .gitignore?

Comment (joshpetit) [reply .idea/.gitignore:1]: My mistake, that would be fine!

File: create_shortcut.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ Terminal=false
 Categories=Office;TextEditor;Utility
 Type=Application
 Icon=${WORKING_DIR}/icon.png
-StartupWMClass=Lotion
+StartupWMClass=$1
```
Comment (puneetsl) [inline create_shortcut.sh:18]: Awesome change! thanks

**Comment by puneetsl** [comment]
I like the changes, just the `.idea` folder can be added to .gitignore (or removed from the PR). I'll merge as soon as you make that change. 
And I really appreciate the contributions, Thanks a lot

**Comment by puneetsl** [comment]
Perfect! Merging

**Pull request merged**